tests/contrib/storage/test_linux_ceph.py

```diff
@@ -434,6 +480,34 @@ def test_erasure_pool_create(self, get_osds, erasure_profile):
                   'application', 'enable', 'test', 'unknown'])
         ])
 
+    @patch.object(ceph_utils, 'kv')
+    @patch.object(ceph_utils, 'get_erasure_profile')
+    @patch.object(ceph_utils, 'get_osds')
+    def test_erasure_pool_create_autoscaler(self,
+                                            get_osds,
+                                            erasure_profile,
+                                            kv):
+        db = MagicMock()
+        kv.return_value = db
+        db.get.return_value = ['pg_autoscaler']
```

Comment:
This unit test appears to do uneccessary mocking now that you query the state directly from Ceph with subprocess in ``enabled_manager_modules``